### PR TITLE
WIP : First step to fix #104

### LINKED ISSUE #104: Implement true Alignak reload
I would like for Alignak-arbiter to reload (config) without changing PID.

### PR DESCRIPTION
**Pull request by jremond**
add SIGHUP generation in init script and signal catching in daemon.py

### CODE REVIEW

**Comment by Seb-Solon** [comment]
For now it's fine. Now we "only" miss the function reload_config (or something like that)  and call it when logging "Reloading configuration"

PS : Pep8 is sad , see https://travis-ci.org/Alignak-monitoring/alignak/jobs/71862174#L905, not a big deal.

File: bin/init.d/alignak [hunk 1]
```diff
@@ -504,14 +504,17 @@ do_reload_() {
         if [ $checkrc -ne 0 ]; then
            return 1
         fi
+        pid=$(getmodpid "$mod")
+        # send SIGHUP signal to reload configuration
+        kill -1 $pid
```
Comment (gst) [inline bin/init.d/alignak]: please use signal name (-HUP) to kill

**Comment by gst** [comment]
The question is now : what to do exactly when the HUP is received..

**Comment by Seb-Solon** [comment]
Done with #265

**Pull request closed without merging**